cv.tex

```diff
@@ -58,71 +59,252 @@
-\cventry{2004--2005}{Master of sciences}{Acoustics \& Signal Processing}{IRCAM, Paris}{}{}
+\vspace{-0.5cm}
+\cvitem{REPOS\\(2021--)}
+{Repos animal en milieu anthropisé. \underline{Partner}. \emph{MUSE (\textbf{390k€})}
+\begin{itemize}
+\item[$\bullet~$] \emph{Human impact on animal behaviours.} 
+\item[$\bullet~$] \emph{My contribution:} Audio processing for logging animal activity. Data analysis. 
+\end{itemize}}
 
-\cventry{1999--2004}{Engineering school}{Telecom Paris, Paris, France}{}{}{}
+\vspace{-0.5cm}
+\cvitem{multiNODE\\(2021--)}
+{Deep models of animal dynamic behaviour. \underline{Principal Investigator}. \emph{MUSE (\textbf{100k€})}
+\begin{itemize}
+\item[$\bullet~$] \emph{Large-scale analysis of animal tracking data.} 
+\item[$\bullet~$] \emph{Deep learning models for cross-modal dynamic data.} 
+\end{itemize}}
 
-\vspace{-0.3cm}
-\section{Scientific service}
+\vspace{-0.5cm}
+\cvitem{ADSIL\\(2020--)}
+{Écoute intelligente sous-marine avancée. \underline{Partner}. \emph{ANR (\textbf{570k€})}
+\begin{itemize}
+\item[$\bullet~$] \emph{Systems and methods for subaquatic acoustics applied to animal tracking.} 
+\item[$\bullet~$] \emph{My contribution:} deep learning for denoising and modeling dynamic data. 
+\end{itemize}}
 
-\cvitem{Commitees}
-{\vspace{-0.4cm}\begin{itemize}
-\item[$\bullet~$]  IEEE Tech. Committee on Audio \& Acoustics (2016-2021)
-\item[$\bullet~$]  Separation Eval. Campaign (SiSEC) general chair (2016-2018)
+\vspace{-0.5cm}\cvitem{KAMoulox\\(2016-2020)}
+{Online unmixing of large historical archives. \underline{Principal investigator}. \emph{ANR (\textbf{350k€})}
+\begin{itemize}
+\item[$\bullet~$] Track \emph{Technology for the art industry and heritage preservation}
+\item[$\bullet~$] \emph{Audio machine learning for cultural heritage preservation and re-purposing} 
+\item[$\bullet~$] \emph{1 PhD. 2 postdocs. 6 journals, 2 book chapters, 22 conferences.} 
 \end{itemize}}
 \vspace{-0.4cm}
-\cvitem{Reviewing}
-{\vspace{-0.4cm}\begin{itemize}
-\item[$\bullet~$] journals : \emph{SPL, TASLP, TSP, SPM, Elsevier SP, DSP, ...}
-\item[$\bullet~$] conferences: \emph{ICML, NeurIPS, ICASSP, ISMIR, WASPAA, ...}
+\cvitem{Industry\\(2020-2021)}
+{\textbf{Industrial transfer}. I was \underline{managing} the transfer of music processing technology to
+companies in Europe and in the USA, for a total of several hundreds
+of thousands euros. Everything about these is confidential and ruled
+by non­disclosure agreements.}
+
+
+
+\section{Miscellaneous}
+\vspace{-0.1cm}
+\cvitem{languages}
+{\begin{itemize}
+   \item[$\bullet~$] French: \emph{native}
+   \item[$\bullet~$] English: \emph{fluent}
+   \item[$\bullet~$] Luxembourgish: \emph{good understanding + learning to talk}
+   \item[$\bullet~$] Spanish: \emph{conversational}
+   \item[$\bullet~$] German: \emph{school-level. Understanding. Not practiced much.}
+\end{itemize}}
+
+\vspace{-0.1cm}
+\cvitem{computing languages}
+{Strong expertise in Python and MATLAB\begin{itemize}
```

Comment:
You are my next matlab reviewer then 😈

Reply:
😱 I think I forgot all of it actually. Will update